.github/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Changes from release 2025/06 to 2025/07
+1. Bug - Added Warton (EGNO) Positions to Military APC Profile
```

Comment:
```suggestion
1. Bug - Added Warton (EGNO) Frequencies to Military APC Profile
```
Techincally it's the frequency being added rather than the position on the logon drop down :)